src/mesh_operations.cpp

```diff
@@ -38,7 +38,13 @@
 #include <float.h>
 
 #include <console_bridge/console.h>
+
+// Shim so this works on foxy/galactic/rolling
+#if __has_include(<tf2_eigen/tf2_eigen.hpp>)
```

Comment:
Why don't you test for `resource_retriever/retriever.hpp` directly?

Reply:
Because I copied this from somewhere in moveit without reading it :facepalm: 

Fixed that.